snapshot/snapshot.go

```diff
@@ -301,6 +305,7 @@ func (o *snapshotter) Prepare(ctx context.Context, key, parent string, opts ...s
 	//       or not, using the key `remoteSnapshotLogKey` defined in the above. This
 	//       log is used by tests in this project.
 	lCtx := log.WithLogger(ctx, log.G(ctx).WithField("key", key).WithField("parent", parent))
+	log.G(lCtx).WithField(remoteSnapshotLogKey, prepareSucceeded).Debug("preparing snapshot")
```

Comment:
small change: Could we omit `WithField(remoteSnapshotLogKey, prepareSucceeded)` here. It should only be logged after we were able to perform a "remote" mount: https://github.com/awslabs/soci-snapshotter/pull/1036/files#diff-5c6bcd0ff5f76d3d6e311122b52e890cff76233ee29e88a138f074bcc74518e1R318